code/modules/mob/living/carbon/emote.dm

```diff
@@ -11,7 +11,35 @@
 	key_third_person = "blinks"
 	message = "blinks."
 
-/datum/emote/living/carbon/blink_r
+/datum/emote/living/carbon/blink/run_emote(mob/living/carbon/human, params, type_override, intentional)
+	. = ..()
+
+	var/obj/item/bodypart/head/head = human.get_bodypart(BODY_ZONE_HEAD)
+
+
+	if(!head)
+		// dullahans are stupid
+		return
+
+	// Don't blink. Not even for a second.
+	for(var/obj/structure/statue/peanut in view(8, human))
```

Comment:
View is a bad check to do here because it only takes into account the statues that the human can see. This means if a statue can see the human but the human can't see the statue, they will not get counted because you've used the human as the src instead of the location the human is on or the statue themselves.